dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!Doctype html>
```

Comment:
I did not any stuff inside `dist` folder its auto generated based on your template provided in `src/index.html`